### ISSUE TITLE
RNA alphabet for Bio.motif.meme

### ISSUE THREAD

**Issue by zaidurrehman** [biopython/biopython#865]
The latest version of meme tool supports RNA alphabet. The parser for meme in Bio.motif is outdated and only accommodates protein and DNA alphabet. 

Adding a single 'elif' statement to Bio/motifs/meme.py on line 176 would solve the issue;

```
if line == 'ACGT':
    al = IUPAC.unambiguous_dna
elif line == 'ACGU':
    al = IUPAC.unambiguous_rna
else:
    al = IUPAC.protein
```

**Comment by peterjc** [comment]
Can you turn this into a pull request? You can even do this via GitHub by editing the file within Python, but I'd suggest trying http://biopython.org/wiki/GitUsage

Do you have a (small) RNA alphabet example motif file we can use for a new test case?

**Comment by zaidurrehman** [comment]
Sure, I can make the changes and make a pull request. Just give me a day for it.
Here is a sample meme output file with RNA alphabet: 
[meme.txt](https://github.com/biopython/biopython/files/339962/meme.txt)

**Comment by peterjc** [comment]
Fixed via #1421, thank you!
